modules/ossm-installing-primary-remote-single-network-mesh.adoc

```diff
@@ -0,0 +1,140 @@
+// This procedure is used in the following assembly:
+// * install/ossm-multi-cluster-topologies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-installing-primary-remote-single-network-mesh_{context}"]
+= Installing a primary-remote single-network mesh 
+
+Install {istio} in a primary-remote single-network topology on two {ocp-product-title} clusters. 
+
+[NOTE]
+====
+In this procedure, `CLUSTER1` is the East cluster and `CLUSTER2` is the West cluster. 
+====
+
+You can adapt these instructions for a mesh spanning more than two clusters.
+
+.Prerequisites
+
+* You have installed all of the clusters on the same network, and there is direct connectivity between the pods in all of the clusters.
+
+* You have installed the {SMProduct} Operator on all of the clusters that comprise the mesh.
+
+* You have completed "Creating certificates for a multi-cluster mesh". 
+
+* You have completed "Applying certificates to a multi-cluster topology".
```

Comment:
After some more investigation, it seems that for the primary-remote case these steps aren't needed as the trust is applied when creating the remote secret.

The remote istiod uses the same certificated and CA from the primary one.

These steps are only needed for the multi-primary use case.